lib/data/analytics.js

```diff
@@ -154,6 +154,10 @@ const metricsTemplate = {
         "num_failed_entities": {
           "recent": 0,
           "total": 0
+        },
+        "num_updated_entities": {
```

Comment:
I think this name has the potential to be slightly confusing. We're measuring the number of entity updates, where each entity can be counted multiple times (once for each update). However, the name of the metric seems to imply that we're measuring a subset of all entities. Put another way, it's possible for `num_updated_entities` > `num_entities`.

@sadiqkhoja, I'll let you decide whether to use the existing name or a different name like `num_entity_updates`. The name isn't shown in Frontend (text is shown instead), so it's just a question of whether we could end up getting confused internally.

Reply:
Do we want to count "Number of Entities Updated" or "Number of Updates across all Entities"?

Reply:
I'm not sure actually. getodk/central#410 suggests the latter. "Number of Entities Updated" would never exceed "Number of Entities", but I think it'd be interesting if we saw a very high "Number of Updates across all Entities". It'd be nice to have some sense of how often updates happen.